modules/core/src/main/scala/fs2/kafka/TransactionalProducerRecords.scala

```diff
@@ -37,20 +37,20 @@ import fs2.kafka.internal.syntax._
   * commit exactly one offset, then emit a [[ProducerResult]] with the
   * results and specified passthrough value.
   */
-sealed abstract class TransactionalProducerRecords[F[_], G[+_], +K, +V, +P] {
```

Comment:
As far as I can tell, the only point of this `G[_]` was to pass through the type of the wrapped `records` field.